flink-python/pyflink/table/expression.py

```diff
@@ -1609,6 +1609,17 @@ def array_min(self) -> 'Expression':
         """
         return _unary_op("arrayMin")(self)
 
+    def split(self, delimiter) -> 'Expression':
```

Comment:
Please add python test(s) for `split` as well

Reply:
I just support Java one, so I think we don't need add python test(s) for `split`.